modules/tinymce/src/core/main/ts/api/dom/Selection.ts

```diff
@@ -310,11 +310,12 @@ const EditorSelection = (dom: DOMUtils, win: Window, serializer: DomSerializer,
     }
 
     try {
-      if ((selection = getSel()) && !NodeType.isRestrictedNode(selection.anchorNode)) {
+      const selection = getSel();
+      if (selection && !NodeType.isRestrictedNode(selection.anchorNode)) {
         if (selection.rangeCount > 0) {
           rng = selection.getRangeAt(0);
         } else {
-          rng = selection.createRange ? selection.createRange() : doc.createRange();
```

Comment:
Minor code change here, as `selection.createRange()` is an older IE API that no longer exists.